torch/_inductor/scheduler.py

```diff
@@ -2138,6 +2138,7 @@ def get_order(n):
 
     @dynamo_timed
     def codegen(self):
+        device = None
```

Comment:
`device` is specific for each node, you'd better move this into the for loop below?

Reply:
Done